### ISSUE TITLE
Title tags on Blog posts

### ISSUE THREAD

**Issue by jeremylenz** [CunningFolkDev/elevate360_website#180]
_3. This isn’t about the new website, but I was told that we don’t have title tags on our blog posts. Could you add these?_

**Comment by jeremylenz** [comment]
@cunning-folk I wasn't able to recreate the issue exactly - there's definitely a `<title>` tag on each blog post page, as well as on the main Blog page. 

_Hypothesis:_ However, on the main Blog page, the `<meta>` tags for `keywords` and `description` were blank.  This may be what they were referring to.   I looked into it and found that the `seo_title`, `seo_keywords` and `seo_description` fields were missing from the Posts collection.  

_Root cause:_ I went back and looked at the original Pages doc with the SEO copy, and it looks like there wasn't one included for Posts or Blog.  So it was never added.

_Solution:_ I re-added those fields (on elevate, not Wholeview) and copied the copy from a different section.  They will have to edit as they see fit.  The `<title>` and `<meta>` tags should now be showing, both on Post pages and the main Blog page.
